pkg/bpf/map.go

```diff
@@ -224,3 +225,21 @@ func GetMapType(t MapType) MapType {
 	}
 	return t
 }
+
+var commonNameRegexps = []*regexp.Regexp{
+	regexp.MustCompile(`^(cilium_)(.+)_reserved_[0-9]+$`),
+	regexp.MustCompile(`^(cilium_)(.+)_netdev_ns_[0-9]+$`),
+	regexp.MustCompile(`^(cilium_)(.+)_overlay_[0-9]+$`),
+	regexp.MustCompile(`^(cilium_)(.+)_[0-9]+$`),
+	regexp.MustCompile(`^(cilium_)(.+)+$`),
+}
+
+func extractCommonName(name string) string {
```

Comment:
:heart: 

Would be neat to make use of this in the stale map collection handling logic too. But that's an issue for a later PR.